taskcluster/ci/config.yml

```diff
@@ -26,7 +26,7 @@ taskgraph:
         mobile:
             name: "Fenix"
     cached-task-prefix: project.mobile.fenix
-    decision-parameters: 'fenix_taskgraph:get_decision_parameters'
+    decision-parameters: 'fenix_taskgraph.parameters:get_decision_parameters'
```

Comment:
+1 to have a dedicated module rather than baking them in the `__init__`

Reply:
Context: I imported this from https://searchfox.org/mozilla-central/rev/4e228dc5f594340d35da7453829ad9f3c3cb8b58/taskcluster/taskgraph/parameters.py :)